src/main/kotlin/no/skatteetaten/aurora/boober/utils/CollectionUtils.kt

```diff
@@ -43,4 +47,21 @@ fun <T> Collection<T>?.nullOnEmpty(): Collection<T>? {
     }
 
     return this
+}
+
+// function that filters out everything except the lines with the provided keys from a .properties ByteArray
+fun filterProperties(properties: ByteArray, keys: List<String>) : ByteArray {
+    val props = Properties()
```

Comment:
.let er fin her synes jeg Properties().let{ it.load(...)}

Reply:
Jeg mener også at man bør eksplisitt håndtere innesningsfeil her (ugyldig properties-format) og kaste exception med informasjon om hvilken fil det er snakk om og hvorfor det feilet.

Reply:
Enig.